### PR TITLE
Fix status when setting channel members

### PR DESCRIPTION
**Pull request by kleewho**
Additional changes:

* ~~simplifies the types in the internal data classes by removing unnecessary wrapping for channel and uuid.~~ They are required actually ;)
* put types in separate files as some are not related at all, which made it more confusing

### CODE REVIEW

File: src/main/kotlin/com/pubnub/api/endpoints/objects/member/ManageChannelMembers.kt [hunk 1]
```diff
@@ -10,7 +10,6 @@ import com.pubnub.api.models.consumer.objects.member.PNMemberArrayResult
 import com.pubnub.api.models.server.objects_api.ChangeMemberInput
 import com.pubnub.api.models.server.objects_api.EntityArrayEnvelope
 import com.pubnub.api.models.server.objects_api.MemberInput
-import com.pubnub.api.models.server.objects_api.UUIDId
 import com.pubnub.extension.toPNMemberArrayResult
 import retrofit2.Call
 import retrofit2.Response
```
Comment (marcin-cebo) [inline src/main/kotlin/com/pubnub/api/endpoints/objects/member/ManageChannelMembers.kt:15]: Why we have this line:
import com.pubnub.api.models.consumer.objects.member.MemberInput as ConsumerMemberInput
>

**Review by marcin-cebo — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/pubnub/api/endpoints/objects/member/ManageChannelMembers.kt (hunk 1 above)
Comment (kleewho) [reply src/main/kotlin/com/pubnub/api/endpoints/objects/member/ManageChannelMembers.kt:15]: To distinguish two different MemberInput types. One is in `consumer` package and the other is in `server` package

File: src/integrationTest/kotlin/com/pubnub/api/integration/ObjectsIntegrationTest.kt [hunk 2]
```diff
@@ -192,14 +178,110 @@ class ObjectsIntegrationTest : BaseIntegrationTest() {
...
+        pubnub.manageChannelMembers(
+            channel = channel,
+            uuidsToSet = listOf(PNMember.Partial(uuidId = testUuid, status = status)),
+            uuidsToRemove = listOf()
+        ).sync()
+
+        val getAllResult =
+            pubnub.getChannelMembers(channel = channel, includeUUIDDetails = PNUUIDDetailsLevel.UUID).sync()!!.data
+
+        pubnub.manageChannelMembers(
```
[the remark this answers is not in the document]
Comment (kleewho) [reply src/integrationTest/kotlin/com/pubnub/api/integration/ObjectsIntegrationTest.kt:206]: Sure. I'll add it

File: src/integrationTest/kotlin/com/pubnub/api/integration/ObjectsIntegrationTest.kt [hunk 3]
```diff
@@ -192,14 +178,110 @@ class ObjectsIntegrationTest : BaseIntegrationTest() {
...
-                assertEquals(this?.data?.name, expectedName)
-                assertEquals(this?.data?.email, expectedEmail)
-                assertEquals(this?.data?.externalId, expectedExternalId)
-                assertEquals(this?.data?.profileUrl, expectedProfileUrl)
-                assertEquals(this?.data?.custom, expectedCustom)
-            }
+    }
+
+    @Test
+    fun testManageMember() {
```
[the remark this answers is not in the document]
Comment (kleewho) [reply src/integrationTest/kotlin/com/pubnub/api/integration/ObjectsIntegrationTest.kt:192]: IMHO 200+ lines of tests in one file is not that much

File: src/main/kotlin/com/pubnub/api/endpoints/objects/member/ManageChannelMembers.kt (hunk 1 above)
Comment (kleewho) [reply src/main/kotlin/com/pubnub/api/endpoints/objects/member/ManageChannelMembers.kt:17]: I renamed one of them, so it's not required to use alias anymore

**Pull request merged**